scss/_regions/_navigation.scss

```diff
@@ -285,10 +285,83 @@
   a {
     display: block;
     font-weight: bold;
-    margin-left: 18px;
     padding: 6px 9px;
   }
 
+  .navigation__dropdown {
+
+    padding-top: 0;
+
+    @include media($tablet) {
+      padding-top: 12px;
+    }
+
+    ul {
+      visibility: visibile;
+      margin-top: 0;
+      @include media($tablet) {
+        visibility: hidden;
+      }
+
+      li {
+        padding: 0; 
+
+        @include media($tablet) {
+          padding: gutter() 0;
+        }
+      }
+
+    }
+
+    p {
+      visibility: hidden;
+      @include media($tablet) {
+        visibility: visible;
+        color: #fff;
+        padding-left: 8px;
+        padding-right: 8px;        
+      }      
+    }
+
+  }
+
+  .navigation__dropdown.clicked {
+    background-color: #fff;
+    padding-top: 0;
+    padding-bottom: 0; 
+    border-radius: 4px;
```

Comment:
This should be `$lg-border-radius` from our [variables](https://github.com/DoSomething/neue/blob/dev/scss/_utilities/_variables.scss) rather than a hardcoded pixel value.